api/swagger.yaml

```diff
@@ -1610,6 +1610,13 @@ definitions:
       ID:
         type: "string"
       Version:
+        description: |
+          The version number of the node object. This is required to avoid conflicting writes.
+          For every node object, the version number is incremented each time it is updated. It is required
```

Comment:
@thaJeztah Yes. You can make a definition for it, then reference it with `$ref`. I think this works for any property - even integers and strings and so on if you wanted to de-dupe the descriptions.

@dnephin I think the problem is duplication, not not defining them well.